css/dashboard.css

```diff
@@ -1,3 +1,4 @@
 .app-icon-calendar {
-	background-image: url('../img/calendar-dark.svg');
+	background-image: var(--icon-calendar-white);
```

Comment:
this will not work because this variable doesnt exist anymore. All you need to do is to add the filter `filter: var(--background-invert-if-dark);` and leave the background image like it was :+1:

Reply:
Thank you for reviewing my patch. It looked like the right approach because of the other widgets on my dashboard had it all done with either inline SVG or CSS variables. However, I've made the changes according to your advice. :+1: